### ISSUE TITLE
[Bug] CORs error when publishing to Gitlab Pages

### ISSUE THREAD

**Issue by lashmore** [chipzoller/hugo-clarity#405]
### I confirm:

- [X] I am running the [extended version of Hugo version 0.91.0 or above](https://github.com/chipzoller/hugo-clarity/blob/master/README.md#prerequisites) to build my site.
- [X] I have read the [Hugo Clarity theme documentation](https://github.com/chipzoller/hugo-clarity/blob/master/README.md).
- [X] I have used the [example site's config files](https://github.com/chipzoller/hugo-clarity/tree/master/exampleSite/config/_default) as reference.
- [X] I have reviewed [other issues in this repository](https://github.com/chipzoller/hugo-clarity/issues) and mine is not recorded.

### Hugo version

hugo v0.92.0+extended darwin/amd64 BuildDate=unknown

### Where is this bug showing up?

In the browser: Hugo builds the site, but something doesn't look right.

### Operating system

macOS

### Browser (if applicable)

Chrome 107.0.5304.87

### Current behavior

The landing page and the About Me page all load fine, but the posts themselves are “Dangerous” when I access via Chrome. 

I saw [this Issue](https://github.com/chipzoller/hugo-clarity/issues/375), but my baseURL never has "www" as part of it, as Gitlab pages have the URL structure "https:/[GITLAB_USER_NAME].gitlab.io/[NAME_OF_REPO]". 

```
Access to fetch at 'https://gitlab.com/oauth/authorize?client_id=5059a88907e1b093c23df47d996183b101a862a5e53e099b563120d1308db2c1&redirect_uri=https://projects.gitlab.io/auth&response_type=code&state=gF-0-JpkGFuAeKM0nMOIKA==&scope=api' (redirected from 'https://lthistlethwaite-prnt.gitlab.io/icons/link.svg') from origin 'https://lthistlethwaite-prnt.gitlab.io/' has been blocked by CORS policy: No 'Access-Control-Allow-Origin' header is present on the requested resource. If an opaque response serves your needs, set the request's mode to 'no-cors' to fetch the resource with CORS disabled.
```

### Expected behavior

I expect the posts / articles to load / be viewed when I click the READ MORE button.

### Steps to reproduce

Go to https://lthistlethwaite-prnt.gitlab.io/brainy-etc/ and click READ MORE on any of the posts. You'll get a Deceptive site ahead from Chrome.

### Relevant log output

```shell
Access to fetch at 'https://gitlab.com/oauth/authorize?client_id=5059a88907e1b093c23df47d996183b101a862a5e53e099b563120d1308db2c1&redirect_uri=https://projects.gitlab.io/auth&response_type=code&state=gF-0-JpkGFuAeKM0nMOIKA==&scope=api' (redirected from 'https://lthistlethwaite-prnt.gitlab.io/icons/link.svg') from origin 'https://lthistlethwaite-prnt.gitlab.io/' has been blocked by CORS policy: No 'Access-Control-Allow-Origin' header is present on the requested resource. If an opaque response serves your needs, set the request's mode to 'no-cors' to fetch the resource with CORS disabled.
```

**Comment by onweru** [comment]
@lashmore https://lthistlethwaite-prnt.gitlab.io/ returns 404, not found

**Comment by lashmore** [comment]
I set my baseURL in the config.toml to: https://lthistlethwaite-prnt.gitlab.io/brainy-etc/, which doesn't have the 404. 

Interestingly, Chrome is no longer throwing the DANGEROUS warning/error, but the post itself is all zoomed in / weird looking. Example post (WIP): https://lthistlethwaite-prnt.gitlab.io/brainy-etc/post/viz/

Note, I made no changes to the deployed version of the site, so I'm confused what caused the change in behavior. Was a javascript file HTTP address in the theme updated recently, maybe?
